src/main/java/com/sintern/api/StudentController.java

```diff
@@ -13,15 +22,40 @@
 @RequestMapping("/students")
 public class StudentController {
     private final StudentService studentService;
+    private final FileService fileService;
 
     @Autowired
-    public StudentController(StudentService studentService) {
+    public StudentController(StudentService studentService, FileService fileService) {
         this.studentService = studentService;
+        this.fileService = fileService;
     }
 
     @PostMapping("/register")
     @ResponseStatus(HttpStatus.OK)
     public void studentRegister(@Valid @RequestBody Student student) {
         studentService.addStudent(student);
     }
+
+    @PostMapping("/upload-cv")
+    @ResponseStatus(HttpStatus.OK)
+    public void uploadCv(@RequestParam("id") UUID studentId, @RequestParam("file") MultipartFile cv) {
+        fileService.save(cv, studentId);
+    }
+
+    @GetMapping("/cv/{studentId}")
```

Comment:
Since the root is `/students/` and here we are accessing a student's CV, the endpoint should be `/students/{studentId}/cv`. We access the list of students, search for a specific student because we want his/her CV. Please change this.